src/components/UpdatesFeed/index.js

```diff
@@ -36,16 +68,29 @@ const UpdatesFeed = ({ setSelectedTab }) => {
 		[data],
 	);
 
-	const handleTabButtonClick = (index) => {
-		setSelectedTab(index);
-	};
-
 	return (
 		<div className={classes.container}>
-			<UpdatesHeader onButtonNavClick={handleTabButtonClick} />
-			{updates.map((update) => (
-				<UpdateCard onButtonNavClick={handleTabButtonClick} key={update.id} {...update} />
-			))}
+			<div className={classes.header}>
+				<div>
+					<div className={classes.line} />
+					<h1 className={classes.headerTitle}>Updates</h1>
+				</div>
+				<Button
+					className={classes.updatesButton}
+					color="primary"
+					variant="text"
+					onClick={() => setSelectedTab(HOME_TAB_KEYS.UPDATES)}
+				>
+					Search all Updates &gt;
+				</Button>
+			</div>
+			<ul className={classes.cardList}>
```

Comment:
Please add an `aria-label` to the `ul`.  Maybe "Updates list"?

Reply:
updated in e791791